nipype/pipeline/engine/nodes.py

```diff
@@ -56,6 +56,15 @@
 logger = logging.getLogger('workflow')
 
 
+class _null_lock(object):
```

Comment:
Wouldn't it be nice to have this stub in `nipype.utils.filemanip`?

Reply:
Except it doesn't do anything. It's certainly not manipulating files. I can put it somewhere else, but as a hidden variable used once, it's not obvious to me that there's a better spot than the file it's used in.